src/abstractdataframe/abstractdataframe.jl

```diff
@@ -581,6 +603,8 @@ missing values, the vector is wrapped in a call to `skipmissing`: custom
 functions must therefore support such objects (and not only vectors), and cannot
 access missing values.
 
+Metadata: this function drops all metadata.
```

Comment:
I considered this, but I concluded that it can be added later if users complain - exactly as you say.

Also 95% of time the result of `describe` is not kept but just displayed immediately so metadata is not that relevant here.